app/models/persistent_volume.rb

```diff
@@ -1,4 +1,6 @@
 class PersistentVolume < ContainerVolume
   acts_as_miq_taggable
+  include NewWithTypeStiMixin
+  has_one :persistent_volume_claim, :dependent => :destroy
```

Comment:
Controversial naming for now. Let's leave it to `persistent_volume_claim` and have a discussion.